### PR TITLE
Issue #457 Fix incorrect spacing on locations page section titles and CSS refactoring

### PR DESCRIPTION
**Pull request by anprok**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

## Steps for review
#457 
- [x] Open locations page and filter by Camps or Facilites only, verify that title is at same position as YMCA initially

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build220 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build220 |
| Installation process | http://install.openy-dev.ffwua.com/build220/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER_COMPOSER/220/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build220/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build220/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build220/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build220/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build220/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL http://ci.openymca.org/build220/profiles/contrib/openy/build/reports/behat and failed screenshots http://ci.openymca.org/build220/profiles/contrib/openy/build/reports/behat/screenshots

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build221 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build221 |
| Installation process | http://install.openy-dev.ffwua.com/build221/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER_COMPOSER/221/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build221/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build221/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build221/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build221/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build221/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL http://ci.openymca.org/build221/profiles/contrib/openy/build/reports/behat and failed screenshots http://ci.openymca.org/build221/profiles/contrib/openy/build/reports/behat/screenshots

File: themes/openy_themes/openy_rose/scss/modules/_location.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      .label-for-field-location-fax,
+      .label-for-field-location-phone {
+        text-transform: lowercase;
+        display: inline;
+      }
+      .field-location-fax,
+      .field-location-phone {
+        display: inline;
+      }
+      h1 {
```
Comment (AlexNetman) [inline themes/openy_themes/openy_rose/scss/modules/_location.scss]: I guess that was force color for word "YMCAs" when it was h1.
Now we have word "YMCAs" as h2. So no need for this code.

**Comment by AlexNetman** [comment]
Tested. Related issue about paddings is appear to be fixed.

Also during development was created new SCSS file with styles. Some of those are redundant and can be removed. See https://github.com/ymcatwincities/openy/pull/928/files#r163272726

Nice to copy all other related styles into unified place.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build223 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build223 |
| Installation process | http://install.openy-dev.ffwua.com/build223/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER_COMPOSER/223/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build223/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build223/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build223/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build223/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build223/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL http://ci.openymca.org/build223/profiles/contrib/openy/build/reports/behat and failed screenshots http://ci.openymca.org/build223/profiles/contrib/openy/build/reports/behat/screenshots

**Review by AlexNetman — APPROVED** [review]

**Pull request merged**